pkg/scripts/node.go

```diff
@@ -16,16 +16,19 @@ limitations under the License.
 
 package scripts
 
-const (
-	drainNodeScriptTemplate = `
-sudo KUBECONFIG=/etc/kubernetes/admin.conf \
-    kubectl drain {{ .NODE_NAME }} --ignore-daemonsets --delete-local-data
-`
-
-	uncordonNodeScriptTemplate = `
-sudo KUBECONFIG=/etc/kubernetes/admin.conf \
-    kubectl uncordon {{ .NODE_NAME }}
-`
+import "github.com/MakeNowJust/heredoc/v2"
+
+var (
+	hostnameScript = heredoc.Doc(`
+		fqdn=$(hostname -f)
+		[ "$fqdn" = localhost ] && fqdn=$(hostname)
+		echo "$fqdn"
+	`)
+
+	drainNodeScriptTemplate = heredoc.Doc(`
+		sudo KUBECONFIG=/etc/kubernetes/admin.conf \
+		kubectl drain {{ .NODE_NAME }} --ignore-daemonsets --delete-local-data
+	`)
```

Comment:
Can we also use the Kubernetes API for draining the node, like we do for cordoning?

Reply:
Yeah, but it requires additional 200 lines of code for no good reason. Someday maybe 🤷‍♂️.